sys/include/net/ethertype.h

```diff
@@ -38,6 +38,22 @@ extern "C" {
 #define ETHERTYPE_NDN           (0x8624)    /**< NDN Protocol (http://named-data.net/) */
 #define ETHERTYPE_IPV6          (0x86dd)    /**< Internet protocol version 6 */
 #define ETHERTYPE_6LOENC        (0xa0ed)    /**< 6LoWPAN encapsulation */
+
+/**
+ * @defgroup    net_ethertype_custom_config Custom ethertype
+ * @ingroup     config
+ * @brief       Custom ethertype definition
+ * @warning     Do not use an IANA assigned number (see https://www.iana.org/assignments/ieee-802-numbers/ieee-802-numbers.xhtml) here
```

Comment:
```suggestion
 * @warning     Do not use a IANA assigned number (see https://www.iana.org/assignments/ieee-802-numbers/ieee-802-numbers.xhtml) here
```

But depends on if you pronounce it "yana" or "I.A.N.A", I guess 😅. I think however the first pronunciation is the established one.

Reply:
Ok, I wasn't quite sure here :D.